scripts/tests/sanitycheck/test_reporting_testsuite.py

```diff
@@ -0,0 +1,168 @@
+#!/usr/bin/env python3
+# Copyright (c) 2020 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+'''
+This test file contains testcases for reporting functionality of Testsuite class of sanitycheck
+'''
+import sys
+import os
+import xml.etree.ElementTree as ET
+import csv
+from collections import defaultdict
+from mock import MagicMock
+import pytest
+
+ZEPHYR_BASE = os.getenv("ZEPHYR_BASE")
+sys.path.insert(0, os.path.join(ZEPHYR_BASE, "scripts/sanity_chk"))
```

Comment:
would be nice some kind of check whether the env var exists or not

Reply:
We had that test earlier, but removed that as the testcases wont even run if the ZEPHYR_BASE is not set, so the need for env var test is eliminated.